### PR TITLE
Create TowerOfHanoi.py

### PR DESCRIPTION
**Pull request by codingCapricorn**
Tower of Hanoi using Python

### CODE REVIEW

File: TowerOfHanoi.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+def TowerOfHanoi(n , source, destination, auxiliary):
+	if n==1:
+		print ("Move disk 1 from source",source,"to destination",destination)
+		return
+	TowerOfHanoi(n-1, source, auxiliary, destination)
+	print ("Move disk",n,"from source",source,"to destination",destination)
+	TowerOfHanoi(n-1, auxiliary, destination, source)
+		
+# Driver code
+n = 4
```
Comment (NitkarshChourasia) [inline TowerOfHanoi.py:12]: Use 
```python
if __name__ == "__main__":
    # Function to execute.

File: TowerOfHanoi.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Recursive Python function to solve the tower of hanoi --
+
```
Comment (NitkarshChourasia) [inline TowerOfHanoi.py:2]: Short and simple.

**Review by NitkarshChourasia — APPROVED** [review]
Better if implemented some more things.

File: TowerOfHanoi.py (hunk 1 above)
Comment (rein14) [reply TowerOfHanoi.py:12]: this is good practice

Comment (NitkarshChourasia) [reply TowerOfHanoi.py:12]: Yes. Indeed.

**Pull request merged**